pkg/handlers/tdf.go

```diff
@@ -91,7 +103,72 @@ func (h Handler) EncryptBytes(tdfType string, unencrypted []byte, attrValues []s
 	}
 }
 
-func (h Handler) DecryptBytes(toDecrypt []byte, disableAssertionCheck bool) (*bytes.Buffer, error) {
+func correctKeyType(alg sdk.AssertionKeyAlg, key interface{}, public bool) (interface{}, error) {
```

Comment:
The only allowed `key` type is `string`, why accept `interface` here instead of forcing the cast earlier?